docker/Dockerfile.benchmark

```diff
@@ -23,6 +23,11 @@ ARG LOCAL_BENCHMARK_SERVE_FOLDER
 
 ENV PYTHONUNBUFFERED TRUE
 
+# Workaround from https://github.com/NVIDIA/nvidia-docker/issues/1632
+RUN apt-key del 7fa2af80
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/3bf863cc.pub
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64/7fa2af80.pub
+
```

Comment:
[docker/Dockerfile.benchmark](https://github.com/pytorch/serve/pull/1627/files#diff-761111a87515028a7e5c334b926a2299484ef91217ae0a169ce5872937d72a8c) should be deprecated due benchmark automate deprecation.

Reply:
Done